src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -228,7 +228,7 @@ window.qBittorrent.DynamicTable ??= (() => {
 
             const onDrag = function(el, event) {
                 if (this.currentHeaderAction === "resize") {
-                    let width = this.startWidth + (event.page.x - this.dragStartX);
+                    let width = this.startWidth + (event.event.pageX - this.dragStartX);
```

Comment:
Not a typo. The `.event` is the native event object from the browser, it is stored within the `event` object from mootools.